CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change history for ui-plugin-eusage-reports
 
+## [2.3.0](https://github.com/folio-org/ui-plugin-eusage-reports/tree/v2.3.0) (2022-10-25)
+
+* Replace babel-eslint with @babel/eslint-parser ([UIPER-93](https://issues.folio.org/browse/UIPER-93))
+* Prop-types is incorrectly listed as a peer-dependency ([UIPER-98](https://issues.folio.org/browse/UIPER-98))
```

Comment:
Could you please rephrase (and any other changelog entries where it applies) to say what has changed, rather than what the problem was? In this case something like
> * Move `prop-types` from peer-dependency to regular dependency. Fixes UIPER-98.